extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1304,23 +1303,67 @@ export class Bind {
 
   /**
    * Print out the current state in the console.
-   * @param {string=} opt_expression
+   * @param {(!Element|string)=} opt_elementOrExpr
    * @private
    */
-  printState_(opt_expression) {
-    if (opt_expression) {
-      if (typeof opt_expression !== 'string') {
-        user().error(TAG, 'Invalid JSON expression. Example: ' +
-            'AMP.printState("foo.bar") to print current value of "foo.bar".');
-        return;
+  debugPrintState_(opt_elementOrExpr) {
+    if (opt_elementOrExpr) {
+      if (typeof opt_elementOrExpr === 'string') {
+        const value = getValueForExpr(this.state_, opt_elementOrExpr);
+        user().info(TAG, value);
+      } else if (opt_elementOrExpr.nodeType === Node.ELEMENT_NODE) {
```

Comment:
nit, nodeType check here uses '===' but a dev assert for nodeType in line 653

`node.nodeType == Node.DOCUMENT_NODE`

 uses '==', can we be consistent and just use one. '==' should be fine.

Reply:
Done.